Source/WebCore/crypto/cocoa/CryptoKeyECMac.cpp

```diff
@@ -312,16 +304,15 @@ RefPtr<CryptoKeyEC> CryptoKeyEC::platformImportSpki(CryptoAlgorithmIdentifier id
     index += bytesUsedToEncodedLength(keyData[index]); // Read length
     if (keyData.size() < index + sizeof(IdEcPublicKey))
         return nullptr;
-    if (memcmp(keyData.data() + index, IdEcPublicKey, sizeof(IdEcPublicKey)))
+    if (!equalSpans(keyData.subspan(index), std::span { IdEcPublicKey }))
```

Comment:
This looks risky, are we sure that `keyData.subspan(index)` has the same size as `IdEcPublicKey`? The previous code would behavior differently if the sizes were different, since it didn't check the size.

`keyData.subspan(index, IdEcPublicKey.size())` would be safer for e.g., unless you're confident it'd fine as-is

Reply:
Oh, good point! Somehow I implicitly assumed that equalSpans would pick the smaller size, but that's obviously not the case.